lib/foreman/renderer/scope/macros/base.rb

```diff
@@ -78,11 +78,41 @@ def default_template_url(template, hostgroup)
                     :host => host, :port => port)
           end
 
+          def load_hosts(search: '', includes: nil)
```

Comment:
Do we want to have this in the base scope and available in all templates? I think it makes sense to move this to a dedicated scope for reports.

Reply:
I thought this would be useful everwhere, e.g. in host's kickstart template, you may want to render a config file based on data of other hosts, e.g. setting up a gateway, /etc/hosts etc